testapps/SimpleApp/SimpleApp.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
IMO there is some value to test all entry points to view compilation. There are three: `buildMultiTargeting/ViewCompilation.targets`, `build/net461/ViewCompilation.targets` and `build/netcoreapp2.0/ViewCompilation.targets`. It's good to ensure all of these work correctly when they are the "entry point" import for view precompilation, aka the outer-build of a project.